module/config.mjs

```diff
@@ -2461,7 +2471,7 @@ patchConfig("conditionTypes", "label", { since: "DnD5e 3.0", until: "DnD5e 3.2"
  */
 DND5E.conditionEffects = {
   noMovement: new Set(["exhaustion-5", "grappled", "paralyzed", "petrified", "restrained", "stunned", "unconscious"]),
-  halfMovement: new Set(["exhaustion-2", "prone"]),
```

Comment:
Prone does not half the speed. Crawling makes 1 foot movement cost 1 extra foot. So in difficult terrain crawling 1 foot costs 3 feet of movement (see PHB). So a prone creature with a speed 60 feet can crawl 20 feet in difficult terrain. If instead the rule was that the speed was halved while prone, a creature with a speed of 60 could only crawl 15 feet (30 feet after halving, then 2 feet per 1 foot makes 15 feet).